authzed/api/v1/core.proto

```diff
@@ -76,6 +76,15 @@ message ZedToken {
   } ];
 }
 
+// Cursor is used to provide resumption of listing between calls to APIs
+// such as LookupResources.
+message Cursor {
+  string token = 1 [ (validate.rules).string = {
+    min_bytes : 1,
```

Comment:
Can we set a max bytes here? Users are going to ask for how much storage they need to store a cursor.

Reply:
Sure